client/package.json

```diff
@@ -11,6 +11,8 @@
     "bootstrap": "^4.5.3",
     "flow-bin": "^0.137.0",
     "graphql": "^15.4.0",
+    "moment": "^2.29.1",
+    "prettier": "^2.2.1",
```

Comment:
this should be in the devDepedencies section because this is not needed for the production code; it's only used for local development.